Ai-spec-driven-development-markdown-best-practices.html

```
@@ -0,0 +1,2657 @@
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="progress-bar" id="progressBar"></div>
+    <button class="back-to-top" id="backToTop" aria-label="ページ上部に戻る" type="button">
+      ↑
+    </button>
+
+    <div class="mobile-topbar">
+      <button class="hamburger" id="hamburgerBtn" aria-label="目次を開く" aria-expanded="false">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)